src/modules/Builder/components/PackageList/PackageList.jsx

```diff
@@ -2,14 +2,66 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 
 import Blackmagic from "api/rpc/blackmagic";
+import * as API from "api/http/images";
 
 import PackagesTable from "./components/PackagesTable/PackagesTable";
 
 export default class PackageList extends Component {
+    static formatDeviceTitle(device) {
+        let title = "";
+        if (device.name) {
+            title = `${title} ${device.name}`;
+        }
+        if (device.generation) {
+            title = `${title} ${device.generation}`;
+        }
+        if (device.model) {
+            title = `${title} ${device.model}`;
+        }
+
+        return title.trim();
+    }
+
     constructor(props) {
         super(props);
 
+        this.state = {
+            basePackagesUrl: "",
```

Comment:
Let's set `null` by default.

Reply:
if set `null` in console we will see `Failed prop type: The prop `basePackagesUrl` is marked as required in `PackagesTable`, but its value is null.`

Reply:
Let's make basePackagesUrl not requeired and `null` by default.